### PR TITLE
Adding a note on the repo to redirect to cdi guide

### PR DESCRIPTION
**Pull request by evelinec**
The MP-intro guide is being archived. 
Adding a note on the repo to indicate and link to the cdi-intro guide.

### CODE REVIEW

**Comment by sishida** [comment]
Looks good! We usually try to avoid using self-referential language, such as saying "this guide," but in this context I'm not sure how to work around it.

File: README.adoc [hunk 1]
```diff
@@ -17,6 +17,9 @@
 :source-highlighter: prettify
 = Creating a MicroProfile application
 
+[.hidden]
+NOTE: This guide is deprecated. View the introductory guide to MicroProfile in https://openliberty.io/guides/cdi-intro.html[Injecting dependencies into microservices].
+
```
Comment (yeekangc) [inline README.adoc]: NOTE: This guide is replaced by https://openliberty.io/guides/rest-intro.html[Creating a RESTful web service
] and https://openliberty.io/guides/cdi-intro.html[Injecting dependencies into microservices] and has been archived. Use those guides instead.

Comment (yeekangc) [reply README.adoc]: @sishida, please review the above.

Comment (evelinec) [reply README.adoc]: Updated.

**Comment by evelinec** [comment]
@chirp1 can you help us take a quick look at this PR in terms of the text?

File: README.adoc [hunk 2]
```diff
@@ -17,6 +17,10 @@
 :source-highlighter: prettify
 = Creating a MicroProfile application
 
+[.hidden]
+NOTE: This guide is replaced by https://openliberty.io/guides/rest-intro.html[Creating a RESTful web service
```
Comment (chirp1) [inline README.adoc]: How about: The _Creating a MicroProfile application_ guide has been archived and is replaced by the https://openliberty.io/guides/rest-intro.html[Creating a RESTful web service] guide and the https://openliberty.io/guides/cdi-intro.html[Injecting dependencies into microservices] guide.

**Pull request merged**